Model/Adminhtml/Checkout.php

```diff
@@ -0,0 +1,283 @@
+                'full_name' => $shippingAddress->getName(),
+                'first' => $shippingAddress->getFirstname(),
+                'last' => $shippingAddress->getLastname()
+            ],
+            'address'=>[
+                'line1' => $shippingAddress->getStreetLine(1),
+                'line2' => empty($shippingAddress->getStreetLine(2)) ? null : $shippingAddress->getStreetLine(2),
+                'city' => $shippingAddress->getCity(),
+                'state' => empty($shippingAddress->getRegionCode()) ? $this->getRegionCode($shippingAddress->getRegionId()) : $shippingAddress->getRegionCode(),
+                'zipcode' => $shippingAddress->getPostcode(),
+                'country' => $shippingAddress->getCountryId()
+            ],
+            'email' => $shippingAddress->getEmail(),
+            'phone_number' => $shippingAddress->getTelephone()
+        ];
+
+        $billingAddress = $order->getBillingAddress();
+        $billingObject = [
+            'name' => [
+                'full_name' => $billingAddress->getName(),
+                'first' => $billingAddress->getFirstname(),
+                'last' => $billingAddress->getLastname()
+            ],
+            'address'=>[
+                'line1' => $billingAddress->getStreetLine(1),
+                'line2' => empty($billingAddress->getStreetLine(2)) ? null : $billingAddress->getStreetLine(2),
+                'city' => $billingAddress->getCity(),
+                'state' => empty($billingAddress->getRegionCode()) ? $this->getRegionCode($billingAddress->getRegionId()) : $billingAddress->getRegionCode(),
+                'zipcode' => $billingAddress->getPostcode(),
+                'country' => $billingAddress->getCountryId()
+            ],
+            'email' => $billingAddress->getEmail(),
+            'phone_number' => $billingAddress->getTelephone()
+        ];
+
+        // Prepare Affirm Checkout Data
+        $shippingAmount = $order->getShippingAmount();
+        $taxAmount = $order->getTaxAmount();
+        $total = $order->getGrandTotal();
+        $data = [
+            'shipping' => $shippingObject,
+            'billing' => $billingObject,
+            'merchant' => [
+                'user_confirmation_url' => $this->urlHelper->getUrl('telesales/payment/confirm'),
+                'user_cancel_url' => $this->urlHelper->getUrl('telesales/payment/cancel'),
+                'user_decline_url' => $this->urlHelper->getUrl('telesales/payment/decline'),
+                'user_confirmation_url_action' => self::METHOD_POST,
+                'public_api_key' => $this->getPublicApiKey()
+            ],
+            'metadata' => [
+                'platform_type' => $this->productMetadata->getName() . self::PLATFORM_TYPE_APPEND,
+                'platform_version' => $this->productMetadata->getVersion() . ' ' . $this->productMetadata->getEdition(),
+                'platform_affirm' => $this->moduleResource->getDbVersion('Astound_Affirm'),
```

Comment:
I think this is grabbing the core affirm version I think we should grab the telesales version

Reply:
Should we denote with a prefix or something that it's the telesales to avoid any ambiguity around whether the platform_affirm value refers to telesales version and not core version?